client/team_role_assignment.go

```diff
@@ -0,0 +1,82 @@
+package client
```

Comment:
all relevant resources are now using this API code.
Deleted the rest of the files (above).